p11-kit/test-generate-keypair.sh

```diff
@@ -10,6 +10,9 @@ test "${abs_top_builddir+set}" = set || {
 : ${P11_MODULE_PATH="$abs_top_builddir"/.libs}
 
 setup() {
+	LC_ALL=C
+	export LC_ALL
```

Comment:
It might make more sense to set this globally, e.g., in `p11_kit_tests_env` in Meson and `AM_TESTS_ENVIRONMENT` in Autotools?

Reply:
Let me try...